src/components/Project/ProjectHeader/GnosisSafeBadge.tsx

```diff
@@ -21,17 +32,90 @@ export function GnosisSafeBadge({ href }: { href: string }) {
     )
   }
 
-  return (
-    <Tooltip
-      placement="bottom"
-      title={
-        <Trans>
-          This project is owned by a Safe.{' '}
-          <Link href={href}>See transactions</Link>.
-        </Trans>
-      }
-    >
-      <Icon component={SafeIcon} />
-    </Tooltip>
-  )
+  function DefaultBadge() {
```

Comment:
**suggestion** its risky to define components within other components, can cause rendering issues. Can we move all these components outside, into their own scope?

Reply:
Good call, done